### DISCUSSION TITLE
Prevent auto-complete from writing until return?

### DISCUSSION THREAD

**Discussion by exastone** [helix-editor/helix#5833] [Q&A]
is there a way to prevent auto-complete suggestions from writing to the buffer until I hit enter?

**Comment by CptPotato** [comment]
To my knowledge there isn't. But I think this would be nice to have as an option.

**Reply by exastone** [reply]
yeah, it drives me a little nuts that it writes to the buffer immediately. Sometime I want to peek at a suggestion, next-thing i know i'm doing hand acrobatics trying to delete everything that just got spewed out.
I'll open a feature request if enough people care
